### PR TITLE
Cleanup for 2.2 rec

### PR DESCRIPTION
**Pull request by michael-n-cooper**
removes change markers for new sc, and a term that is no longer referenced

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wcag/pull/3307.html" title="Last updated on Jun 7, 2024, 3:40 PM UTC (aa2912e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wcag/3307/63d28b8...aa2912e.html" title="Last updated on Jun 7, 2024, 3:40 PM UTC (aa2912e)">Diff</a>

### CODE REVIEW

**Comment by fstrr** [comment]
Updated PR to merge in changes that have leap-frogged it.

**Comment by alastc** [comment]
@iadawn I think this is defunct now? Some of it we've done separately, but should the "(new)" markers be removed?

File: guidelines/terms/22/encloses.html [hunk 1]
```diff
@@ -1,7 +1,3 @@
-<dt class="new"><dfn id="dfn-enclose" data-lt="enclose">encloses</dfn></dt>
-<dd class="new">
-	<p class="change">New</p>
-   					
-   <p>solidly bounds or surrounds</p>
-   				
+<dt><dfn id="dfn-enclose" data-lt="enclose">encloses</dfn></dt>
+  <p>solidly bounds or surrounds</p>
```
Comment (patrickhlauke) [inline guidelines/terms/22/encloses.html:2]:
```suggestion
  <dd><p>solidly bounds or surrounds</p>
```

**Comment by mbgower** [comment]
The TR was published with the "New" content in place. In 2.1, "New" was retained after publishing. The TF is satisfied to retain this text. The removal of "Encloses" has already been achieved.

**Pull request closed without merging**